MekHQ/src/mekhq/campaign/parts/equipment/MissingEquipmentPart.java

```diff
@@ -187,7 +184,9 @@ public boolean isAcceptableReplacement(Part part, boolean refit) {
 		if(part instanceof EquipmentPart) {
 			EquipmentPart eqpart = (EquipmentPart)part;
 			EquipmentType et = eqpart.getType();
-			return type.equals(et) && getTonnage() == part.getTonnage() && part.getStickerPrice().equals(newPart.getStickerPrice());
+			return type.equals(et) && getTonnage() == part.getTonnage()
+                    && size == eqpart.getSize()
```

Comment:
Should the two == comparisons be bracketed? Asking as that's part of our coding style guide I've never been certain on for return values

Reply:
I usually add them. I missed it here.